README.md

```diff
@@ -35,6 +35,12 @@ Having problems or solved a problem? Check out the Islandora google groups for a
 * [Islandora Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora)
 * [Islandora Dev Group](https://groups.google.com/forum/?hl=en&fromgroups#!forum/islandora-dev)
 
+### Known issues
+
+In August 2018, a bug in the Book MODS ingest form was identified and fixed. The form was set to READ all instances of any MODS identifier element, but to CREATE and UPDATE identifiers with the 'type="isbn"' attribute. 
+When editing an object with this form, the form would create a new copy of every identifier element regardless of attributes, which may have led useres to remove those identifiers. 
+Any objects edited using this form before the August 2018 fix should be reviewed to ensure their metadata is correct.
```

Comment:
Perhaps something like the following would be even more precise, while leaving space for future extensions to the _Known Issues_ sub-sub-section?
```
### Known Issues
* **Islandora Book MODS Form**: In August 2018, a bug in the stock Book MODS Form was identified and fixed.
    The form's **ISBN** element did READ every instance of a MODS `<identifier>` element, irrespective of its `type` attribute, while it did CREATE and UPDATE `<identifier>` elements with the `type="isbn"` attribute. As a result, when editing an object with this form, the form would show an **ISBN** text field for every `<identifier>` element regardless of attributes. This may have led users to remove or edit the mislabelled identifiers.
    Any objects edited using this form before the August 2018 fix should, therefore, be reviewed to ensure their metadata is correct.
```
yielding
> ### Known Issues
> * **Islandora Book MODS Form**: In August 2018, a bug in the stock Book MODS Form was identified and fixed.
    The form's **ISBN** element did READ every instance of a MODS `<identifier>` element, irrespective of its `type` attribute, while it did CREATE and UPDATE `<identifier>` elements with the `type="isbn"` attribute. As a result, when editing an object with this form, the form would show an **ISBN** text field for every `<identifier>` element regardless of attributes. This may have led users to remove or edit the mislabelled identifiers.
    Any objects edited using this form before the August 2018 fix should, therefore, be reviewed to ensure their metadata is correct.

But, of course, this is merely a suggestion...

Reply:
Works for me.